src/simtools/db/db_handler.py

```diff
@@ -24,20 +25,47 @@
 # The above comment is because pylint does not know that DatabaseHandler.db_client is subscriptable
 
 
+jsonschema_db_dict = {
+    "$schema": "https://json-schema.org/draft-07/schema#",
```

Comment:
I seem to get a warning from the `jsonschema` library `DeprecationWarning: The metaschema specified by $schema was not found. Using the latest draft to validate, but this will raise an error in the future.` Do you get this too?

Reply:
Thanks a lot for that - I didn't see the warning when I worked on the code (I can only see it when I run pytest with the "-s" option.

I've updated the jsonschema version throughout the code (in some schema files unrelated to the DB).

Fixed also one issue noticed now when running tests without a local '.env' file (which we anyway shouldn't; but let's keep the code robust)